setup.py

```diff
@@ -75,7 +75,7 @@
     # > implementing the protocol, and integrations for both blocking I/O and
     # > for different asynchronous frameworks.
     # https://web.archive.org/web/20241206000411/https://www.freedesktop.org/wiki/Software/DBusBindings/
-    install_requires=["aiomqtt>=2,<3", "jeepney>=0.8,<0.9"],
+    install_requires=["aiomqtt>=2,<3", "jeepney>=0.9,<1.0"],
```

Comment:
```suggestion
    install_requires=["aiomqtt>=2,<3", "jeepney>=0.8,<1.0"],
```
let's keep the compatibility with jeepney v0.8, as we do not use any features introduced in v0.9